dataplane/forwarding/fwdaction/actions/mirror.go

```diff
@@ -96,8 +96,8 @@ func (m *mirror) Process(packet fwdpacket.Packet, counters fwdobject.Counters) (
 			if err != nil {
 				return fmt.Errorf("actions: mirror actions failed to get output port, err %v", err)
 			}
-			packet.Logf(fwdpacket.LogDebugMessage, "transmitting packet to %q", out.ID())
-			packet.Logf(fwdpacket.LogDesc, fmt.Sprintf("%v: Transmit %v", m.ctx.ID, out.ID()))
+			packet.Log().Info("transmitting packet", "port", out.ID())
+			packet.Log().WithValues("context", m.ctx.ID, "port", out.ID())
```

Comment:
Should these two lines be reversed? Is WithValues is printing anything? https://pkg.go.dev/github.com/go-logr/logr#Logger.WithValues

Reply:
this is intentional, WithValues mutates the log sink so that subsequent logs have those values